pkg/service/legacy/service.go

```diff
@@ -156,3 +152,7 @@ func bucketKeyToShardLink(key string) (ipld.Link, error) {
 	}
 	return cidlink.Link{Cid: shard}, nil
 }
+
+func isAbsoluteURL(u url.URL) bool {
+	return u.Scheme != "" && u.Host != ""
+}
```

Comment:
there is a `func (*url.URL) IsAbs() bool` function in the standard library, would that work for your use case? Interestingly, it's implementation only checks that `Scheme` is not empty.

Reply:
I think that'll do.